channel.go

```diff
@@ -197,6 +198,8 @@ func (ch *Channel) call(req message, res ...message) error {
 			// error on the Connection.  This indicates we have already been
 			// shutdown and if were waiting, will have returned from the errors chan.
 			return ErrClosed
+		case <-time.After(ch.connection.Config.Heartbeat):
+			return ErrChannelOpTimeout
```

Comment:
is there any cleanup there might be required here? to handle rpc responses delivered after this select loop exits due to a timeout?